docs/deployment/content/remote-local-setup.yaml

```diff
@@ -24,11 +24,14 @@ spec:
           ln -s ~/.cache/apk /etc/apk/cache
           apk add apache2-utils bash bash-completion bat bind-tools conntrack-tools coreutils curl \
                   diffutils dnsmasq docker-bash-completion findutils fzf g++ gawk gcompat git      \
-                  git-prompt go grep helm inotify-tools jq k9s less lsof make mandoc mc moreutils  \
+                  git-prompt grep helm inotify-tools jq k9s less lsof make mandoc mc moreutils     \
                   mount ncurses neovim parallel procps sed strace tar tcpdump tmux tmux-doc tzdata \
                   util-linux vim wget yq
           mkdir -p ~/.cache/wget
           cd ~/.cache/wget
+          echo golang          && wget -N "https://go.dev/dl/$(curl -sL https://golang.org/VERSION?m=text | awk 'NR==1{print}').linux-amd64.tar.gz" && tar -C /usr/local -xzf go1.*.linux-amd64.tar.gz \
```

Comment:
Could this also run in an arm64 linux environment?

Reply:
Yes, it could, but the go binary is only one of the many amd64 binaries used in the remote local setup. The challenge to extend this to arm64 as well is that we would need to prepare a test environment, which we don't have at the moment as far as I know. 

If it would be okay with you, I'd rather propose we address this in a follow-up PR because that will take a bit longer than just fixing the current remote local setup. Would you be fine with that?